src/main/java/org/jvnet/hudson/plugins/port_allocator/PortAllocator.java

```diff
@@ -190,7 +190,7 @@ private void checkPortNumbers(String ports) throws FormException {
 
         public Pool getPoolByName(String poolName) throws PoolNotDefinedException {
             for (Pool p : pools) {
-                if (p.name.toUpperCase().equals(poolName.toUpperCase())) {
+                if (p.name.equals(poolName)) {
```

Comment:
It's a breaking change BTW. I think the author just wanted to have a case-insensitive comparison, so I would use http://docs.oracle.com/javase/6/docs/api/java/lang/String.html#equalsIgnoreCase(java.lang.String)